src/Resources/assets/js/onboardingWizard/main.js

```diff
@@ -0,0 +1,3 @@
+import onboardingWizard from './OnboardingWizard';
```

Comment:
here it should be
`import OnboardingWizard from './OnboardingWizard';`
`const tour = new onboardingWizard();`
and then, every usage of `onboardingWizard` be used with camelCase. What I mean is that class should be PascalCase, but declared instances - camelCase